backend/api/Startup.cs

```diff
@@ -277,13 +278,13 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IApiVers
                 config.MapHealthChecksUI(
                     setup =>
                     {
-                        setup.UIPath = "/healthchecks-ui"; // this is ui path in your browser
-                        // setup.ApiPath = "/api";
-                        setup.ResourcesPath = "/ui/resources";
-                        setup.WebhookPath = "/hooks";
-                        setup.UseRelativeResourcesPath = false;
-                        setup.UseRelativeApiPath = false;
-                        setup.UseRelativeWebhookPath = false;
+                        setup.UIPath = this.Configuration.GetValue<string>("HealthChecksUI:UIPath"); // this is ui path in your browser
```

Comment:
I suspect all of this is not required if it's in the configuration, but I can't find any good documentation.